src/modules/tools/endstops/Endstops.cpp

```diff
@@ -705,10 +720,12 @@ void Endstops::process_home_command(Gcode* gcode)
     // on some systems where 0,0 is bed center it is nice to have home goto 0,0 after homing
     // default is off for cartesian on for deltas
     if(!is_delta) {
-        // NOTE a rotary delta usually has optical or hall-effect endstops so it is safe to go past them a little bit
-        if(this->move_to_origin_after_home) move_to_origin(haxis);
         // if limit switches are enabled we must back off endstop after setting home
```

Comment:
Are the endstops disabled during the move_to_origin?  If I home to Y_Max what would happen if while moving to Y=0 it went too far perhaps because someone set the config incorrectly... would the Y-Min limit switch stop it or has it not been re-enabled yet?